<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTagDialog</class>
 <widget class="QDialog" name="EditTagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editing Tag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="4">
      <widget class="QFontComboBox" name="fontCB">
       <property name="currentFont">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="frontColorPB">
       <property name="text">
        <string>Front Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="backgroundColorPB">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="numberL">
       <property name="text">
        <string>Number :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="numberSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="shapeL">
       <property name="text">
        <string>Shape :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="shapeCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Cercle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tirangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditTagDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditTagDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
